docs/guidance/map/framework_agreements.md

```diff
@@ -81,23 +81,24 @@ The two extensions should be declared in the package metadata:
 :language: json
 ```
 
-The following guidance describes how to model the different stages of a framework agreement procedure in OCDS.  
+The following guidance describes how to model the different stages of a framework agreement procedure in OCDS. (The guidance does not cover the modeling of the contract (using the `contracts` array), because this information about a framework agreement is modeled the same as for other types of contracts.)
```

Comment:
```suggestion
The following guidance describes how to model the different stages of a framework agreement procedure in OCDS. (It does not describe how to model the contract (using the `contracts` array), because the modeling is the same for a framework agreement as for other types of contracts.)
```